pkg/apis/nfd/validate/validate_test.go

```diff
@@ -1,10 +1,27 @@
+/*
+Copyright 2023 The Kubernetes Authors.
```

Comment:
24?

Reply:
The file was created in 2023 (by you, I think 😅) but the license header was missing. So I put the year of the original contribution here. Makes sense?